public/javascripts/global.js

```diff
@@ -184,37 +185,27 @@ function fetchQuiz(event) {
           api_team_id = team.data('team-id'),
           league = team.data('league'),
           rb_team_id = team.data('id'),
-          team = team.data('team');
+          team_name = team.data('team'),
+          list_id = team.data('list-id'),
+          list_name = team.data('list-name');
     }
     else{
       var team_id = $('#teams option:selected').val();
     }
 
-    var data = {
-      api_team_id: api_team_id, 
-      league: league, 
-      rb_team_id: rb_team_id, 
-      team_name: team
-    };
-
-    AjaxCreateQuiz(data);
-}
-
-function AjaxCreateQuiz(data){
```

Comment:
think we need to add this method back. It got lost somewhere w/ the merge conflicts